### PR TITLE
add resource alicloud_slb_ca_certificate support.

### PR DESCRIPTION
**Pull request by wzb56**
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud% TF_ACC=1 go test ./alicloud -v  -sweep=cn-beijing -sweep-run=alicloud_slb_ca_certificate
2018/10/23 18:05:02 [DEBUG] Running Sweepers for region (cn-beijing):
2018/10/23 18:05:04 [INFO] Skipping Slb CA Certificate: CAcer (CAcer)
2018/10/23 18:05:04 Sweeper Tests ran:
	- alicloud_slb_ca_certificate
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	1.368s
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud%
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud% TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudSlbCACertificate -timeout=300m -count=1
=== RUN   TestAccAlicloudSlbCACertificate_import
--- PASS: TestAccAlicloudSlbCACertificate_import (2.91s)
=== RUN   TestAccAlicloudSlbCACertificate_basic
--- PASS: TestAccAlicloudSlbCACertificate_basic (1.51s)
PASS
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	4.487s

### CODE REVIEW

**Comment by wzb56** [comment]
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud% TF_ACC=1 go test ./alicloud -v  -sweep=cn-beijing -sweep-run=alicloud_slb_ca_certificate
^@2018/10/24 00:19:08 [DEBUG] Running Sweepers for region (cn-beijing):
^R
2018/10/24 00:19:10 [INFO] Skipping Slb CA Certificate: CAcer (CAcer)
2018/10/24 00:19:10 Sweeper Tests ran:
	- alicloud_slb_ca_certificate
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	2.131s
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud% TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudSlbCACertificate -timeout=300m
=== RUN   TestAccAlicloudSlbCACertificate_import
^@--- PASS: TestAccAlicloudSlbCACertificate_import (2.50s)
=== RUN   TestAccAlicloudSlbCACertificate_basic
--- PASS: TestAccAlicloudSlbCACertificate_basic (1.96s)
PASS
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	4.504s

File: alicloud/resource_alicloud_slb_ca_certificate.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+func resourceAlicloudSlbCACertificateRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*connectivity.AliyunClient)
+	slbService := SlbService{client}
+
+	caCertificate, err := slbService.describeSlbCACertificate(d.Id())
+	if err != nil {
+		if IsExceptedError(err, SlbCACertificateIdNotFound) || NotFoundError(err) {
+			d.SetId("")
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_slb_ca_certificate.go]: I think `IsExceptedError(err, SlbCACertificateIdNotFound)` is redundant and please remove it.

File: alicloud/resource_alicloud_slb_ca_certificate.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	slbService := SlbService{client}
+
+	return resource.Retry(5*time.Minute, func() *resource.RetryError {
+		request := slb.CreateDeleteCACertificateRequest()
+		request.CACertificateId = d.Id()
+		_, err := slbService.client.WithSlbClient(func(slbClient *slb.Client) (interface{}, error) {
+			return slbClient.DeleteCACertificate(request)
+		})
+		if err != nil {
+			if IsExceptedError(err, SlbCACertificateIdNotFound) || NotFoundError(err) {
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_slb_ca_certificate.go]: I think `IsExceptedError(err, SlbCACertificateIdNotFound)` is redundant and please remove it.

Comment (wzb56) [reply alicloud/resource_alicloud_slb_ca_certificate.go]: slbClient.DeleteCACertificate(request) may not return NotFoundError

**Comment by wzb56** [comment]
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud% TF_ACC=1 go test ./alicloud -v  -sweep=cn-beijing -sweep-run=alicloud_slb_ca_certificate
2018/10/24 17:09:59 [DEBUG] Running Sweepers for region (cn-beijing):
^@2018/10/24 17:10:00 [INFO] Skipping Slb CA Certificate: CAcer (CAcer)
2018/10/24 17:10:00 Sweeper Tests ran:
	- alicloud_slb_ca_certificate
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	1.402s
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud% TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudSlbCACertificate -timeout=300m

=== RUN   TestAccAlicloudSlbCACertificate_import
--- PASS: TestAccAlicloudSlbCACertificate_import (3.81s)
=== RUN   TestAccAlicloudSlbCACertificate_basic
--- PASS: TestAccAlicloudSlbCACertificate_basic (2.08s)
PASS
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	(cached)
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud%
zhongbo.wzb@scala:~/go/src/github.com/terraform-providers/terraform-provider-alicloud% TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudSlbCACertificate -timeout=300m -count=1
=== RUN   TestAccAlicloudSlbCACertificate_import
--- PASS: TestAccAlicloudSlbCACertificate_import (2.06s)
=== RUN   TestAccAlicloudSlbCACertificate_basic
--- PASS: TestAccAlicloudSlbCACertificate_basic (1.87s)
PASS
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	3.992s

File: alicloud/resource_alicloud_slb_ca_certificate.go (hunk 1 above)
Comment (wzb56) [reply alicloud/resource_alicloud_slb_ca_certificate.go]: yes, it's redundant.

**Pull request merged**